vms/event/templates/event/create.html

```diff
@@ -198,17 +206,17 @@
                     </div>
                 {% else %}
                     <div class="form-group">
-                        <label class="col-md-2 control-label">{% trans "Venue" %}</label>
+                        <label class="col-md-2 control-label">Venue</label>
```

Comment:
Also can you please reason the removal of ```trans``` VMS supports French, there we need this and to support other languages in future. 🚢

Reply:
I'll do it!